google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -148,39 +155,90 @@ class Paragraph:
+    documentai_block: documentai.Document.Page.Block
     text: str
+    paragraphs: List[Paragraph]
 
 
-@dataclasses.dataclass
-class FormField:
-    """Represents a wrapped documentai.Document.Page.FormField.
+def _table_wrapper_from_documentai_table(
+    documentai_table: documentai.Document.Page.Table, text: str
+) -> Table:
+    r"""Returns a Table.
+
+    Args:
+        documentai_table (documentai.Document.Page.Table):
+            Required. A documentai.Document.Page.Table.
+        text (str):
+            Required. UTF-8 encoded text in reading order
+            from the document.
+
+    Returns:
+        Table:
+            A Table.
 
-    Attributes:
-        documentai_formfield (google.cloud.documentai.Document.Page.FormField):
-            Required. The original google.cloud.documentai.Document.Page.FormField object.
-        field_name (str):
-            Required. The form field name
-        field_value (str):
-            Required. The form field value
     """
 
-    documentai_formfield: documentai.Document.Page.FormField
-    field_name: str
-    field_value: str
+    header_rows = _table_rows_from_documentai_table_rows(
+        table_rows=list(documentai_table.header_rows), text=text
+    )
+    body_rows = _table_rows_from_documentai_table_rows(
+        table_rows=list(documentai_table.body_rows), text=text
+    )
+
+    return Table(
+        documentai_table=documentai_table, body_rows=body_rows, header_rows=header_rows
+    )
+
+
+def _get_hocr_bounding_box(
+    element_with_layout: ElementWithLayout,
+    dimension: documentai.Document.Page.Dimension,
+) -> str:
+    r"""Returns a hOCR bounding box string.
+
+    Args:
+        element_with_layout (ElementWithLayout):
+            Required. an element with layout fields.
+        dimension (documentai.Document.Page.Dimension):
+            Required. Page dimension.
+
+    Returns:
+        str:
+            hOCR bounding box sring.
+    """
+    if element_with_layout.layout.bounding_poly.vertices:
+        min_x, min_y = (
+            element_with_layout.layout.bounding_poly.vertices[0].x,
+            element_with_layout.layout.bounding_poly.vertices[0].y,
+        )
+        max_x, max_y = (
+            element_with_layout.layout.bounding_poly.vertices[2].x,
+            element_with_layout.layout.bounding_poly.vertices[2].y,
+        )
+        return f"bbox {int(min_x)} {int(min_y)} {int(max_x)} {int(max_y)}"
+    min_x, min_y = (
+        element_with_layout.layout.bounding_poly.normalized_vertices[0].x,
+        element_with_layout.layout.bounding_poly.normalized_vertices[0].y,
+    )
+    max_x, max_y = (
+        element_with_layout.layout.bounding_poly.normalized_vertices[2].x,
+        element_with_layout.layout.bounding_poly.normalized_vertices[2].y,
+    )
+    return f"bbox {int(min_x * dimension.width)} {int(min_y * dimension.height)} {int(max_x * dimension.width)} {int(max_y * dimension.height)}"
```

Comment:
I think the "switch" nature is more obvious if we put second half of the code in an `else` statement.  Also the code here is largely duplicated, why not refactor this out into a little function that can be easily tested, for example

```
def _xy_from_vertices(vertices: ..., min: bool = True) -> Tuple[int, int]:
    ... 
```

Reply:
wil do.